src/Core/SalesChannelRequest.php

```diff
@@ -8,14 +8,20 @@
 final class SalesChannelRequest
 {
     public const ATTRIBUTE_IS_SALES_CHANNEL_REQUEST = '_is_sales_channel';
+    public const ATTRIBUTE_IS_STORE_API_REQUEST = '_is_store_api';
 
     public const ATTRIBUTE_THEME_ID = 'theme-id';
     public const ATTRIBUTE_THEME_NAME = 'theme-name';
     public const ATTRIBUTE_THEME_BASE_NAME = 'theme-base-name';
 
     public const ATTRIBUTE_SALES_CHANNEL_MAINTENANCE = 'sw-maintenance';
 
-    public const ATTRIBUTE_SALES_CHANNEL_MAINTENANCE_IP_WHITLELIST = 'sw-maintenance-ip-whitelist';
+    /**
+     * @deprecated tag:v6.8.0 - Will be removed, use \Shopware\Core\SalesChannelRequest::ATTRIBUTE_SALES_CHANNEL_MAINTENANCE_IP_WHITELIST instead.
+     */
+    public const ATTRIBUTE_SALES_CHANNEL_MAINTENANCE_IP_WHITLELIST = self::ATTRIBUTE_SALES_CHANNEL_MAINTENANCE_IP_WHITELIST;
+
+    public const ATTRIBUTE_SALES_CHANNEL_MAINTENANCE_IP_WHITELIST = 'sw-maintenance-ip-whitelist';
```

Comment:
if we are going to deprecate the old one anyway, can we name the new one like this?
```suggestion
    public const ATTRIBUTE_SALES_CHANNEL_MAINTENANCE_IP_ALLOWLIST = 'sw-maintenance-ip-whitelist';
```